assignment-2/7s.md

```diff
@@ -1,2 +1,296 @@
+*On compensation* A formal promotion should lead to higher pay in a straightforward and transparent way.
+
+##### Skills
+
+> Skills are the special, distinctive competencies of the organization, both in the people and the
+> organization itself. What is the organization good at, and what special skills and competencies
+> are found its people?
+
+What is the organization good at?
+According to [17]:
+- Spotify has better social features than its rivals, with the ability to add friends and see their playlists and listening, as well as a built-in messaging system to ping music back and forth. Creating and sharing your own playlists is easy too.
+- Another strength is the system for following musicians’ profiles and getting notifications when they have new music out. A recently introduced Concerts feed shows you upcoming, nearby gigs by artists you’ve listened to, which is useful.
+- It has good support for devices beyond the desktop and mobile: it’s available on Sony’s PlayStation 4, Sonos hi-fis and through Google’s Chromecast and Amazon’s Fire TV, while its mobile app plays nicely with Apple TV too.
+- Spotify’s free tier has audio advertising, but is missing offline listening and – on mobile – the ability to pick tracks on-demand, although you can play an artist’s catalogue on shuffle mode. Pay £9.99 a month, and you get full access (including offline), no ads, and a higher-quality streams option.
+- Where Spotify falls down is its family plan, which is more expensive than rivals. You can add up to four extra accounts for £4.99 each, but that means you’ll pay £29.95 a month for a family of five compared to Apple Music and Google Play’s £14.99 for six.
+- Besides all this facts actually there are a couple of easy things to say where Spotify is very good at, like Spotify seems to be good at keeping their organization dynamic, focusing on their community rather to an hierarchical structure.
+
+##### Style
+
+> Style deals mainly with the leadership approach of the top managers in the organization. How do
+> the leaders interface with subordinates and others in the organization, and how do members
+> interact with each other? Style also incorporates the climate or culture of the organization, which
+> can be expressed in terms of job titles, space allocation, workplace architecture, artifacts, and
+> policies.
+
+How do the leaders interact with subordinates and others in the organization?
+According with Spotify [12]:
+- They have an Agile engineering culture, which means they like servent leaders rather than process masters.
+- Autonomy that every member team has over their work have evolve to a high rate percentage of team members satisfaction and they keep investing on it. 
+- Autonomy seems also to keep high motivation levels [12 and 13], teams are fast, and their minimize de dependency from and to, other teams.
```

Comment:
"seems also to" => "also seems to" 

"de" => "the" (I do this one a lot too...haha)